app/controllers/signup_controller.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```

Comment:
I think we don't need the signup on this PR, you can remove it after testing

Reply:
and for the signup, I think it should be `UsersController` as in this controller, we want to create a new user

Reply:
Make sense, since signup is not a part of this PR